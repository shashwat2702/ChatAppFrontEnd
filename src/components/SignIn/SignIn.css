div {
    text-align: center;
}
body {
    background-color: #333333;
}