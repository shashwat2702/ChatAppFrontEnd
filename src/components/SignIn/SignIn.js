import React, { Component, Fragment } from 'react';
import { Link } from 'react-router-dom';
import './SignIn.css';
import Header from '../shared/Header/Header';
import Footer from '../shared/Footer/Footer';

export default class SignInPage extends Component {
  render() {
    return (
      <Fragment>
      <Header/>
      <div>
        <Link to="/">
          Click here to go Home Page
        </Link>
      </div>
      <Footer/>
      </Fragment>
    )
  }
}
