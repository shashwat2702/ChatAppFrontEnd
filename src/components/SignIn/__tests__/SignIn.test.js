import React from 'react';
import renderer from 'react-test-renderer';
import SignIn from '../SignIn';

jest.mock('react-router-dom');

describe('SignIn snapshot',() => {
    it('should render correctly',() => {
        const SignInSnap = renderer.create(<SignIn/>);
        const SignInJson = SignInSnap.toJSON();
        expect(SignInJson).toMatchSnapshot();
    });
});