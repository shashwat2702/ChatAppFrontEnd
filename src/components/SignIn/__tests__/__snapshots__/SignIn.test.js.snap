// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn snapshot should render correctly 1`] = `
Array [
  <div
    class="pageHeader"
  >
    Welcome to Home Page
  </div>,
  <div />,
]
`;
